doc/eip.md

```diff
@@ -12,28 +12,58 @@ requires: EIP-155, EIP-1014, EIP-7702
+This ERC defines a permissionless and deterministic deployment mechanism for a universal CREATE2 factory contract with a fixed cross-chain address (0xC0DE207acb0888c5409E51F27390Dad75e4ECbe7) and known bytecode. By leveraging EIP-7702 for stateless delegation and EIP-1014's CREATE2 (0xf5) opcode, this standard enables trustless, reproducible contract deployments to deterministic addresses across all EVM-compatible chains—without relying on preinstalls, secret keys, or chain-specific infrastructure.
 
 ## Motivation
 
 Ensuring that contracts share the same address and code on multiple chains is a hard problem. It is typically done by having a known CREATE2 factory contract at a specific address that can further deterministically deploy new contracts using the `CREATE2 (0xf5)` opcode.
 
-However, there is a bootstrapping problem: how do you get a CREATE2 factory contract with specific address and code? There are currently three main approaches to this problem:
+However, there is a bootstrapping problem: how do you get a CREATE2 factory contract with a specific address and code? 
 
-1. Use Nick's method to randomly generate a signature for a transaction **without** [EIP-155](./eip-155.md) replay protection that deploys the CREATE2 factory. Nick's method ensures that there is no known private key for account that deploys the CREATE2 factory, meaning that the resulting contract will have deterministic address and code on all chains. This is the strategy used by [Arachnid/deterministic-deployment-proxy](https://github.com/Arachnid/deterministic-deployment-proxy), one of the most widely used CREATE2 factory contracts. However, this method comes with some downsides:
-   - It does not work on chains that only accept EIP-155 replay protected transactions.
-   - It is sensitive to changes in gas parameters on the target chain since the gas price in the deployment transaction is sealed, and without a private key, a new one cannot be signed.
+### Existing Solutions
+
+There are currently three main approaches to this problem:
+
+#### 1. Nick's method
+
+Use Nick's method to randomly generate a signature for a transaction **without** [EIP-155](./eip-155.md) replay protection that deploys the CREATE2 factory. Nick's method ensures that there is no known private key for an account that deploys the CREATE2 factory, meaning that the resulting contract will have a deterministic address and code on all chains. This strategy is used by [Arachnid/deterministic-deployment-proxy](https://github.com/Arachnid/deterministic-deployment-proxy), one of the most widely used CREATE2 factory contracts.
+
+**Downsides**:
+
+   - It does not work on chains that only accept EIP-155 replay-protected transactions.
+   - It is sensitive to changes in gas parameters on the target chain since the gas price in the deployment transaction is sealed, and a new one cannot be signed without a private key.
```

Comment:
nit: I wouldn't indent these lines. I don't think it makes a difference for Markdown rendering.